Tests/PackageSyntaxTests/InMemoryGitRepository.swift

```diff
@@ -0,0 +1,482 @@
+/*
```

Comment:
This is a pared-down version of what's in SPMTestSupport, it should be possible to remove it entirely